app/src/behaviors/behavior_toggle_layer.c

```diff
@@ -84,6 +85,7 @@ static const struct behavior_driver_api behavior_tog_driver_api = {
 #define KT_INST(n)                                                                                 \
```

Comment:
```suggestion
#define TG_INST(n)                                                                                 \
```
(need to fix line 94 also)